index.html

```diff
@@ -970,9 +910,62 @@ <h2>Methods</h2>
         </dt>
         <dd>
           <p>
-            The {{setSelectedCandidatePair}} method attempts to change the selected candidate pair. If successful, data will be sent
-            on the provided candidate pair. When this method is invoked, the [= user agent =] MUST run the steps to [= change the
-            selected candidate pair =].
+            The {{setSelectedCandidatePair}} method attempts to select a different candidate pair to send data
+            over. If successful, data will be sent on the provided candidate pair.
+            It is meant to be called after the application defers the {{nomination}} of a candidate pair
+            by cancelling the {{RTCIceTransport/icecandidatepairnominate}} event.
+          <p>
+          <p>
+            When this method is invoked, the [= user agent =] MUST run the following steps:
+          </p>
+          <ol class="algorithm">
+            <li>
+              <p>
+                Let |connection:RTCPeerConnection| be the {{RTCPeerConnection}} object associated with [=this=].
```

Comment:
nit: "associated with this" sounds like we're referring to something close to "this" but other than "this", when we are in fact referring to "this". Should this be "Let |connection:RTCPeerConnection| be [=this=] {{RTCPeerConnection}}."?

Or if [=this=] is already a variable you can refer to, we don't need |connection|, we can just s/connection/[=this=]

Reply:
Sorry ignore my comment we're not on the peer connection!